ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/management-eapi-vrf-name-use_mgmt_interface_vrf.yml

```diff
@@ -0,0 +1,21 @@
+---
+type: l3leaf
+
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 10.10.10.0/24
+    vtep_loopback_ipv4_pool: 10.10.10.0/24
+    bgp_as: 65042
+  nodes:
+    - name: management-eapi-vrf-name-use_mgmt_interface_vrf
+      loopback_ipv4_address: 10.42.42.42
+      id: 12
+
+management_eapi:
+  vrfs:
+    # Assigning default value (use_mgmt_interface_vrf) as user defined to test negative scenario.
```

Comment:
```suggestion
    # Assigning default value (`use_mgmt_interface_vrf`) as user-defined to test negative scenario because AVD is currently ignoring the error for backward-compatibility. This will change in AVD 6.0
```